warrior/Actions/SeleniumActions/browser_actions.py

```diff
@@ -182,10 +182,8 @@ def browser_launch(self, system_name, browser_name="all", type="firefox",
                     output_dict[browser_fullname] = browser_inst
                     if "url" in browser_details and browser_details["url"]\
                             is not None:
-                        result, url = self.browser_object.check_url(browser_details["url"])
-                        if result == True:
-                            result = self.browser_object.go_to(url,
-                                                               browser_inst)
+                        self.browser_object.check_url(browser_details["url"])
```

Comment:
The bug is url navigation has nothing to do check_url(). So check_url should not impact warrior attempting to navigate to the specified url. 
Check url just checks the function and no navigation is performed in this function.
Check_url is based on urllib2 module, which can validate public urls with http/https as schemes. 

The current purpose of check_url is to send warning/info/error to the user with appropriate message to the user.